Portfolio.html

```diff
@@ -4,11 +4,76 @@
     <meta charset="UTF-8">
     <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
     <link rel="stylesheet" type="text/css" href="Portfolio.css">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="Media-Query.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Rob Reeves Portfolio</title>
 </head>
-<body>
-
 
+<body>
+    <nav class="main-nav">
+        <div class="nav-button-left">
+            <a class="nav-text" href="#C1">ABOUT ME</a>
+            <a class="nav-text" href="#C3">CONTACT</a>
+        </div>
+        <div class="nav-button-right">
+            <a class="nav-text" href="#C2">THREE PEDALS ONLY</a>
+        </div>
+    </nav>
+        <div class="page-header">
+            <div class="main-header-wrapper">
+                <div class="main-header-centre">
+                    <h1>ROB REEVES</h1>
+                    <div class="sub-heading-wrapper">
+                        <p class="sub-heading">------- THIS WEBSITE IS ALL ABOUT ME! ------</p>
+                    </div>
+                    <div class="sub-heading-wrapper2">
+                        <p class="sub-heading2">There are lots of interesting things about me on this website</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    <div class="main-content" id="C1">
+        <div class="main-content-wrapper">
+            <h2>------ ABOUT ME ------</h2>
+            <p class="main-content-text">
+                Lots of interesting stuff about me, all of which is indeed very interesting and it will be so interesting that you will not fall asleep reading it all I promise. Are you not entertained yet?
```

Comment:
Seems to be some p tags with content on same line, & some with content on it's own line.  Might be better to be consistent?

Reply:
done.